integrations/acquisition/covidcast/test_db.py

```diff
@@ -29,6 +29,28 @@ def _find_matches_for_row(self, row):
                 results[table] = None
         return results
 
+    def test_insert_or_update_with_nonempty_load_table(self):
+        # make rows
+        a_row = self._make_placeholder_row()[0]
+        another_row = self._make_placeholder_row(time_value=self.DEFAULT_TIME_VALUE+1, issue=self.DEFAULT_ISSUE+1)[0]
+        # insert one
+        self._db.insert_or_update_bulk([a_row])
+        # put something into the load table
+        self._db._cursor.execute(f'''
+            INSERT INTO {self._db.load_table}
+            (source, `signal`, geo_type, geo_value, time_type, time_value, issue, `lag`, value_updated_timestamp)
+            VALUES
+            ('sr',   'si',     'gt',     'gv',      'tt',      3,          8,     5,     13)''')
```

Comment:
hmmm i kinda wanna find a way to refactor this out just so we don't have to continually mess with it when we make more changes. did you already look at that?

Reply:
twas indeed quite begrudging.  ive thought about a better way to handle it, but i cant really come up with something that bothers me less.  i also had to do something similar with the AI counter fix method [here](https://github.com/cmu-delphi/delphi-epidata/blob/c890712333cbfa2a519bb2644db0517a869b2b78/src/acquisition/covidcast/database.py#L131)